MdePkg/Include/Protocol/Tcg2Protocol.h

```diff
@@ -23,10 +23,6 @@ typedef struct tdEFI_TCG2_VERSION {
   UINT8    Minor;
 } EFI_TCG2_VERSION;
 
-typedef UINT32 EFI_TCG2_EVENT_LOG_BITMAP;

```

Comment:
Please note in the PR description that these moves are transparent to a platform because UefiTcgPlatform.h is already included in Tcg2Protocol.h.